R/dualpathFused.R

```diff
@@ -339,6 +339,15 @@ dualpathFused <- function(y, D, approx=FALSE, maxsteps=2000, minlam=0,
     }
     completepath = FALSE
   }
+  
+  # If we reached the maximum degrees of freedom
+  else if (df[k-1]>maxdf) {
+    if (verbose) {
+      cat(sprintf("\nReached the maximum degrees of freedom (%.3f),",maxdf))
```

Comment:
You should change `%.3f` here to `%i` (and test it out). The former is for floats (and specifies that we should show 3 decimals); the latter is for integers.

Reply:
Ugh, sloppy. Fixed.